src/styles/globals.css

```
@@ -548,6 +548,43 @@
+.chat-html-content details.agent-thinking > summary {
+  padding: 0.4rem 0.75rem;
+  cursor: pointer;
+  font-size: 0.75rem;
+  color: var(--color-text-muted);
+  background-color: var(--color-surface-secondary);
+  list-style: none;
+  display: flex;
+  align-items: center;
+  gap: 0.375rem;
+  user-select: none;
+}
```

Stylelint (17.7.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)